test/debugger/test-debug-break-on-uncaught.js

```diff
@@ -44,7 +44,12 @@ function runScenario(scriptName, throwsInFile, throwsOnLine, next) {
 
   var exceptions = [];
 
-  setTimeout(setupClient.bind(null, runTest), 200);
+
+  child.stderr.on('data', (data) => {
+    if (data.toString().includes('Debugger listening on port')) {
```

Comment:
Assumes that lines correspond with data events.  I'd concatenate the output and scan that.